src/controllers/profileController.js

```diff
@@ -0,0 +1,126 @@
+import { config } from 'dotenv';
+import cloudinary from 'cloudinary';
+import { User } from '../sequelize/models';
+
+config();
+cloudinary.config({
+  cloud_name: process.env.CLOUDINARY_NAME,
+  api_key: process.env.CLOUDINARY_API_KEY,
+  api_secret: process.env.CLOUDINARY_SECRET_KEY,
+});
+
+/**
+ * @description User profile
+ */
+class Profile {
+  /**
+       * @description get user profile
+       * @param {object} req
+       * @param {object} res
+       * @returns {object} return object containing user profile
+       */
+  static async user(req, res) {
+    const userName = req.params.username;
+    try {
+      const userfound = await User.findOne({ where: { userName } });
+      if (!userfound) {
+        return res.status(404).json({
+          message: `Profile for ${userName} not found`,
+        });
+      }
+      res.status(200).json({
+        message: 'Successfully retrieved a user profile',
+        data: {
+          userName: userfound.userName,
+          bio: userfound.bio,
+          image: userfound.image,
+        }
+      });
+    } catch (error) {
+      // console.log(error);
+    }
+  }
+
+  /**
+   * @description fetch all the profiles
+   * @param {object} req
+   * @param {object} res
+   * @returns {object} all available profiles
+   */
+  static async fetchProfiles(req, res) {
+    try {
+      const users = await User.findAll({
+        where: { role: 'user' },
+        attributes: ['userName', 'bio', 'image']
+      });
+      res.status(200).json({
+        message: `${users.length} Users found`,
+        data: users
+      });
+    } catch (err) {
+      // console.log(err);
+    }
+  }
+
+  /**
+       * @description update user profile
+       * @param {object} req
+       * @param {object} res
+       * @returns {object} return object containing updated user profile
+       */
+  static async editProfile(req, res) {
```

Comment:
According to the title of this PR, `#167484603 Fetch all available profiles`, and then this method is to edit profile. Why did you create it?

Reply:
I did not create that method. I just edited the class where that method was created, Raymond created it